mlir/unittests/Analysis/Presburger/IntegerRelationTest.cpp

```diff
@@ -725,3 +725,18 @@ TEST(IntegerRelationTest, addLocalModulo) {
     EXPECT_TRUE(rel.containsPointNoLocal({x, x % 32}));
   }
 }
+
+TEST(IntegerRelationTest, simplify) {
+  IntegerRelation rel =
+      parseRelationFromSet("(x, y)[N]: (2*x + y - 4*N - 3 == 0, 3*x - y - 3*N"
+                           "+ 2 == 0, x + 3*y - 5*N - 8 == 0, x - y + N >= 0)",
+                           2);
```

Comment:
can you split the set description at constraint boundaries (commas) so it's easier to read?